CHANGELOG.md

```diff
@@ -10,26 +10,61 @@ CHANGELOG
 - Add support for Centos 7 ARM.
 - Add support for FSx Lustre DataCompressionType feature.
 - Add validation to prevent using a `cluster_resource_bucket` that is in a different region than the cluster.
+- Make sure slurmd service is not enabled before finalize stage, which will prevent user from unintentionally making compute node available in post-install process.
+- Change to ssh_target_checker.sh syntax that makes the script compatible with pdsh.
```

Comment:
Also in this case instead of what we did, let's talk about the enhancement if it's the case.

Reply:
Removed, the change can be found in cookbook changelog